### DISCUSSION TITLE
Copliot deducted $200 at one time

### DISCUSSION THREAD

**Discussion by agent-1900** [community/community#30367] [Copilot Conversations]
Copliot deducted $200 at one time

**Comment by agent-1900** [comment]
Billing & plans / Payment history

ID                 Date              Payment method       Amount 
0XCPM2DX   2022-08-22    Visa ending in 8562    $100.00
01HZIZUS     2022-08-22    Visa ending in 8562    $100.00

**Reply by mazkdevf** [reply]
You should contact github support

**Reply by strugglebak** [reply]
I am waiting for their feedback

**Reply by strugglebak** [reply]
I got their feedback just now. Fortunately, somebody noticed the matter...

**Comment by strugglebak** [comment]
same problem

**Comment by strugglebak** [comment]
[Billing & plans](https://github.com/settings/billing) / Payment history
ID DatePayment  AmountReceipt
H8295108 2022-08-22 $100.00
65708006 2022-08-22 $100.00

**Comment by strugglebak** [comment]
up

**Comment by strugglebak** [comment]
I got another automatic deduction just few minutes ago with this stupid bug...

**Reply by agent-1900** [reply]
Very speechless and incomprehensible. I submitted the order, but no one handled it

**Reply by chandraawardhana** [reply]
looks like we are experiencing the same thing. but whether your copilot can be used after paying?

**Reply by strugglebak** [reply]
> looks like we are experiencing the same thing. but whether your copilot can be used after paying?

yes, absolutely it works fine with my vscode ide after paying

**Reply by strugglebak** [reply]
> Very speechless and incomprehensible. I submitted the order, but no one handled it

me too, I really want a refund

**Reply by agent-1900** [reply]
me too, I'm just a coder, want a refund

**Reply by chandraawardhana** [reply]
> > looks like we are experiencing the same thing. but whether your copilot can be used after paying?
> 
> yes, absolutely it works fine with my vscode ide after paying

you're lucky, I can't use my current copilot now even though I already paid and my funds have been deducted from paypal

**Reply by chandraawardhana** [reply]
how to use copilot after paying for it? whether paypal funds must be 2x of subscription funds?

**Comment by strugglebak** [comment]
up

**Comment by strugglebak** [comment]
up

**Comment by strugglebak** [comment]
up

**Comment by strugglebak** [comment]
up

**Comment by chh0724-cheng** [comment]
I had the same problem and had to pay again after clicking

**Comment by agent-1900** [comment]
up

**Comment by strugglebak** [comment]
up

**Comment by Akash1134** [comment]
Hi @ai408 , We're sorry that you're facing issues with Copilot billing. The best place to go for answers to account and billing specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them. Thanks!

**Reply by chandraawardhana** [reply]
> Hi @ai408 , We're sorry that you're facing issues with Copilot billing. The best place to go for answers to account and billing specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them. Thanks!

Can you help me ? I have made a payment using paypal and the funds have been deducted but the copilot cannot be used, the statement is that the trial period has expired.

**Reply by agent-1900** [reply]
@Akash1134  I submitted a support ticket yesterday and no response yet.

**Comment by yusuf-mutlu** [comment]
I was charged $300 for copilot subscription. Very stupid bug. I submitted a support ticket yesterday for $200 refund and no response yet.

**Reply by ibelikov** [reply]
same story - got charged $300, opened a support ticket > 30 hours ago, no reply still.

**Reply by strugglebak** [reply]
I got their feedback from Github Support team, finally I could get my refund soon, hope so...

**Comment by agent-1900** [comment]
@strugglebak me too
